object(Closure)#17 (2) {
  ["this"]=>
  object(Zor\Core)#5 (1) {
    ["isDev":"Zor\Core":private]=>
    bool(true)
  }
  ["parameter"]=>
  array(3) {
    ["$throwable"]=>
    string(10) "<required>"
    ["$request"]=>
    string(10) "<required>"
    ["$response"]=>
    string(10) "<required>"
  }
}
PHP Fatal error:  Uncaught ArgumentCountError: Too few arguments to function Zor\Core::Zor\{closure}(), 0 passed and exactly 3 expected in /app/zor/src/Core.php:193
Stack trace:
#0 [internal function]: Zor\Core->Zor\{closure}()
#1 /app/zor/src/Core.php(210): call_user_func(Object(Closure))
#2 {main}

Next Swoole\ExitException: swoole exit. in /app/zor/bin/bin.php:31
Stack trace:
#0 /app/zor/src/Core.php(215): d(Object(Closure))
#1 {main}
  thrown in /app/zor/bin/bin.php on line 31

Fatal error: Uncaught ArgumentCountError: Too few arguments to function Zor\Core::Zor\{closure}(), 0 passed and exactly 3 expected in /app/zor/src/Core.php:193
Stack trace:
#0 [internal function]: Zor\Core->Zor\{closure}()
#1 /app/zor/src/Core.php(210): call_user_func(Object(Closure))
#2 {main}

Next Swoole\ExitException: swoole exit. in /app/zor/bin/bin.php:31
Stack trace:
#0 /app/zor/src/Core.php(215): d(Object(Closure))
#1 {main}
  thrown in /app/zor/bin/bin.php on line 31
Error at file[/app/zor/bin/bin.php] line[31] message:[Uncaught ArgumentCountError: Too few arguments to function Zor\Core::Zor\{closure}(), 0 passed and exactly 3 expected in /app/zor/src/Core.php:193
Stack trace:
#0 [internal function]: Zor\Core->Zor\{closure}()
#1 /app/zor/src/Core.php(210): call_user_func(Object(Closure))
#2 {main}

Next Swoole\ExitException: swoole exit. in /app/zor/bin/bin.php:31
Stack trace:
#0 /app/zor/src/Core.php(215): d(Object(Closure))
#1 {main}
  thrown]
[2018-12-10 09:59:58 *480.4]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: Uncaught ArgumentCountError: Too few arguments to function Zor\Core::Zor\{closure}(), 0 passed and exactly 3 expected in /app/zor/src/Core.php:193
Stack trace:
#0 [internal function]: Zor\Core->Zor\{closure}()
#1 /app/zor/src/Core.php(210): call_user_func(Object(Closure))
#2 {main}

Next Swoole\ExitException: swoole exit. in /app/zor/bin/bin.php:31
Stack trace:
#0 /app/zor/src/Core.php(215): d(Object(Closure))
#1 {main}
  thrown in /app/zor/bin/bin.php on line 31.
[2018-12-10 09:59:58 $465.0]	WARNING	swManager_check_exit_status: worker#4 abnormal exit, status=255, signal=0
[2018-12-10 10:02:31 #464.2]	NOTICE	Server is shutdown now.
[2018-12-10 10:02:31 @484.0]	WARNING	close: socket has already been bound to another coroutine #1.
[2018-12-10 10:02:35 *505.4]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: Uncaught ArgumentCountError: Too few arguments to function Zor\Core::Zor\{closure}(), 0 passed and exactly 3 expected in /app/zor/src/Core.php:193
Stack trace:
#0 [internal function]: Zor\Core->Zor\{closure}()
#1 /app/zor/src/Core.php(210): call_user_func(Object(Closure))
#2 {main}

Next Swoole\ExitException: swoole exit. in /app/zor/bin/bin.php:31
Stack trace:
#0 /app/zor/src/Core.php(215): d(Object(Closure))
#1 {main}
  thrown in /app/zor/bin/bin.php on line 31.
[2018-12-10 10:02:35 $490.0]	WARNING	swManager_check_exit_status: worker#4 abnormal exit, status=255, signal=0
[2018-12-10 10:05:41 *530.5]	ERROR	zm_deactivate_swoole (ERROR 503): Fatal error: Uncaught ArgumentCountError: Too few arguments to function Zor\Core::Zor\{closure}(), 0 passed and exactly 3 expected in /app/zor/src/Core.php:193
Stack trace:
#0 [internal function]: Zor\Core->Zor\{closure}()
#1 /app/zor/src/Core.php(210): call_user_func(Object(Closure))
#2 {main}

Next Swoole\ExitException: swoole exit. in /app/zor/bin/bin.php:31
Stack trace:
#0 /app/zor/src/Core.php(215): d(Object(Closure))
#1 {main}
  thrown in /app/zor/bin/bin.php on line 31.
[2018-12-10 10:05:41 $514.0]	WARNING	swManager_check_exit_status: worker#5 abnormal exit, status=255, signal=0
